src/modules/ticket/reducer.ts

```diff
@@ -0,0 +1,89 @@
+import {Alert} from 'react-native';
+import {Action} from 'redux';
+import {t} from '@translations';
+
+export const BUY_TICKET_REQUEST = 'BUY_TICKET_REQUEST';
+export const BUY_TICKET_SUCCESS = 'BUY_TICKET_SUCCESS';
+export const BUY_TICKET_FAIL = 'BUY_TICKET_FAIL';
+
+export interface TicketState {
+  isLoading: boolean;
+  error: string | null;
+  reservationId: string | null;
+}
+
+export interface BuyTicketRequestAction extends Action {
+  type: typeof BUY_TICKET_REQUEST;
+  payload: string;
+}
+
+export interface BuyTicketSuccessAction extends Action {
+  type: typeof BUY_TICKET_SUCCESS;
+  payload: {
+    reservationId: string;
+  };
+}
+
+export interface BuyTicketFailAction extends Action {
+  type: typeof BUY_TICKET_FAIL;
+  payload: {
+    error: string;
+  };
+}
+
+export type TicketAction =
+  | BuyTicketRequestAction
+  | BuyTicketSuccessAction
+  | BuyTicketFailAction;
+
+const initialState: TicketState = {
+  isLoading: false,
+  error: null,
+  reservationId: null,
+};
+
+export const ticketReducer = (
+  state = initialState,
+  action: TicketAction,
+): TicketState => {
+  switch (action.type) {
+    case BUY_TICKET_REQUEST: {
+      const newState = state;
+
+      newState.isLoading = true;
+      newState.error = null;
+      newState.reservationId = null;
+
+      return newState;
+    }
+    case BUY_TICKET_SUCCESS: {
+      Alert.alert(
```

Comment:
Optionally, we can use destructuring;
i.e
`const { title, message } = t.myBooking.successReservation;

Alert.alert(
    title,
    message,
);`